invenio_rdm_records/config.py

```diff
@@ -338,4 +304,118 @@ def always_valid(identifier):
         "validator": always_valid
     }
 }
-RDM_RECORDS_DOI_DATACITE_FORMAT = "{prefix}/{id}"
+
+
+#
+# Record permission policy
+#
+RDM_PERMISSION_POLICY = None
+"""Override the default record permission policy."""
+
+
+#
+# Search configuration
+#
+RDM_RECORD_FACETS = {
```

Comment:
Intended to be defined here, and only overwritten by advanced users.

Reply:
I was tempted at simply calling it ``RDM_FACETS``. But then there might be ``RDM_AFFILIATIONS_FACETS``?

My preference would be for ``RDM_FACETS`` (similar below).